app/src/components/Project/CreateProject.tsx

```diff
@@ -13,6 +13,7 @@ const CreateProject = (props: ICreateProjectProps) => {
   const [description, setDescription] = useState('')
   const [tags, setTags] = useState<string[]>([])
   const [buttonColor, setButtonColor] = useState('#EFEFEF')
+  const [buttonActiv, setButtonActiv] = useState('disabled')
```

Comment:
If you have two states like button is active and is not active - you should use the boolean type.
But here we can check this condition like `const isButtonActive = title.length > 0`